pallets/liquidity-pools/src/lib.rs

```diff
@@ -348,13 +348,13 @@ pub mod pallet {
 			Ok(())
 		}
 
-		/// Add an AccountId to the set of known connectors, allowing that
-		/// origin to send incoming messages.
-		#[pallet::weight(< T as Config >::WeightInfo::add_connector())]
+		/// Add an AccountId to the set of known liquidity pool instances,
+		/// allowing that origin to send incoming messages.
+		#[pallet::weight(< T as Config >::WeightInfo::add_instance())]
 		#[pallet::call_index(1)]
-		pub fn add_connector(origin: OriginFor<T>, connector: T::AccountId) -> DispatchResult {
+		pub fn add_instance(origin: OriginFor<T>, instance: T::AccountId) -> DispatchResult {
 			T::AdminOrigin::ensure_origin(origin)?;
-			KnownConnectors::<T>::insert(connector, ());
+			KnownInstances::<T>::insert(instance, ());
```

Comment:
We do not need that here anymore, right?

Reply:
Right! I also didn't expect to see this here anymore but thought that Cosmin may have had a reason to keep it. I'll address this in a follow up PR 🙏🏻